### PR TITLE
feat/#a4k5en_#a6k6a5): profile done

### PR DESCRIPTION
**Pull request by asianswagstorm**
Banner is a simple background color. The profile picture is static logo image.

### CODE REVIEW

**Comment by dannylekim** [comment]
Task linked: [CU-a6k6a5 Display User data matching figma design](https://app.clickup.com/t/a6k6a5)

**Comment by dannylekim** [comment]
Task linked: [CU-a4k5en Profile Page [3 POINTS]](https://app.clickup.com/t/a4k5en)

File: src/components/BillCard/LargeBillCard/styles.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import "../../../variables";
 
 div.large__bill {
-  height: 600px;
+  height: 25%;//600px;
```
Comment (alphakennyn) [inline src/components/BillCard/LargeBillCard/styles.scss:4]: remove comment

File: src/components/Sidebar/styles.scss [hunk 2]
```diff
@@ -5,7 +5,7 @@
   position: fixed;
   height: 100%;
   background-color: $secondary-color;
-  width: 302px;
```
Comment (alphakennyn) [inline src/components/Sidebar/styles.scss:8]: change it to 
`max-width: 302px;`

Comment (alphakennyn) [reply src/components/Sidebar/styles.scss:8]: actually, can you not modify `SideBar` styles? I already made some changes on this on my change

File: src/constants/mockObjects.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export const mockUserInfo = {
```
Comment (alphakennyn) [inline src/constants/mockObjects.js:1]: can you store the file under `src/constant/tests/constants/...`

File: src/views/Dashboard/Dashboard.jsx [hunk 4]
```diff
@@ -1,44 +1,58 @@
...
-export default ({history,userInfo}) => {
-  //testing with @testing-library to be learnt later
-  const [activeState, setActiveState] = useState(
-                                                {
-                                                  ...DEFAULT_ACTIVE_STATE,
-                                                  bills: true
-                                                }
-                                              );
-  const [component, setComponent] = useState(<BillDisplay />);
+export default class Dashboard extends Component {
```
Comment (alphakennyn) [inline src/views/Dashboard/Dashboard.jsx:8]: ok im going to pause reviewing. My refactoring changes will definately have conflicts with this change

**Review by alphakennyn — CHANGES_REQUESTED** [review]

**Comment by dannylekim** [comment]
Try to address the code smells btw

File: src/views/Dashboard/Dashboard.jsx (hunk 4 above)
Comment (asianswagstorm) [reply src/views/Dashboard/Dashboard.jsx:8]: This is not the latest commit https://github.com/dannylekim/billsnapFront/blob/feat/%23a4k5en_%23a6k6a5/src/views/Dashboard/Dashboard.jsx

**Pull request closed without merging**